<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="4" total="5" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Phone : Login Icon clicked]]>
    </line>
    <line>
      <![CDATA[Phone :Login button clicked]]>
    </line>
    <line>
      <![CDATA[Login with OTP: Phone number entered]]>
    </line>
    <line>
      <![CDATA[OTP sent]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="51007" started-at="2018-07-20T13:05:38Z" finished-at="2018-07-20T13:06:29Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="51007" started-at="2018-07-20T13:05:38Z" finished-at="2018-07-20T13:06:29Z">
      <class name="com.qa.testcases.TestCswLogin">
        <test-method status="PASS" signature="url_launch()[pri:0, instance:com.qa.testcases.TestCswLogin@4ae82894]" name="url_launch" is-config="true" duration-ms="33246" started-at="2018-07-20T13:05:38Z" finished-at="2018-07-20T13:06:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- url_launch -->
        <test-method status="FAIL" signature="coverStoryOTPLogin()[pri:4, instance:com.qa.testcases.TestCswLogin@4ae82894]" name="coverStoryOTPLogin" duration-ms="17651" started-at="2018-07-20T13:06:11Z" finished-at="2018-07-20T13:06:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.qa.testcases.TestCswLogin.coverStoryOTPLogin(TestCswLogin.java:250)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Phone : Login Icon clicked]]>
            </line>
            <line>
              <![CDATA[Phone :Login button clicked]]>
            </line>
            <line>
              <![CDATA[Login with OTP: Phone number entered]]>
            </line>
            <line>
              <![CDATA[OTP sent]]>
            </line>
          </reporter-output>
        </test-method> <!-- coverStoryOTPLogin -->
      </class> <!-- com.qa.testcases.TestCswLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
